api/SourceBuffer.json

```diff
@@ -563,10 +563,10 @@
               "version_added": null
             },
             "safari": {
-              "version_added": null
+              "version_added": "12.1"
             },
             "safari_ios": {
-              "version_added": false
+              "version_added": "13"
```

Comment:
The other entries have comments saying "Exposed in Mobile Safari on iPad but not on iPhone."

Testing https://mdn-bcd-collector.appspot.com/tests/api/SourceBuffer on an iPad shows a lot more of the API supported, would you mind updating more of it? Or file an issue and leave this false, it's just one of many incorrectly false entries here.

Reply:
Huh, interestingly enough, this API seems to be supported on iPhone too, so I don't know why that note was added.  I'll update this PR accordingly.

Reply:
Does it actually work on the iPhone, or is the interface just accidentally exposed but useless? https://caniuse.com/mediasource also talks about it being supported only on iPad, so a working demo of it on iPhone would be required to convince me that's wrong :)

Reply:
Alright, so I did some more digging around and found that an API this depends on, `MediaSource`, seems to be supported only on iPad.  A `MediaSource` object is required to get a `SourceBuffer` object (see `api.MediaSource.addSourceBuffer`), and the entire API is marked as unsupported on Safari iOS.

Reply:
Updates have been made!